### DISCUSSION TITLE
I'm not getting the spawning logic

### DISCUSSION THREAD

**Discussion by ElweThor** [sbtoonz/Trader_2.0#12] [Q&A]
Readme tells "_You can find this trader randomly spawned in the meadows. Alternatively you can always spawn him using the prefab name **Knarr**_"... now... if I configure "Should Knarr randomly spawn around your world? = false" I could interpret that as it won't "spawn randomly" (thinking it's "times") but "stable" (AKA like Hador: spawned to some (random) location but **remaining there**: not de-spawning/re-spawning at random).
I walked around my map's "mainland" Meadows, finding nothing at all. So I looked at Changelog and found the **find knarr** command. Result was "No instances of knarr found".
ok
I walked back home, logged off, modified the config: "Should Knarr randomly spawn around your world? = true" as, it's written, this way Knarr will use the locationsystem.
Logged back in the game, find knarr... "No instances of knarr found" ç__ç
My suggestion is, at least, to explain better the spawning logic (which, at you, the creator, will seems trivial, but to a non-modder is not) so to correctly config (and finally FIND) the #@£$%!!! knarr.
Am I forced to manually spawn Knarr? (I hope not)
Will he spawn somewhere in the Meadows, sooner or later? (I hope yes)

**Comment by sbtoonz** [comment]
https://github.com/sbtoonz/Trader_2.0/issues/11#issuecomment-1328095655

**Comment by sbtoonz** [comment]
I gotta write the responses in the console a little better etc 

knarr only spawns on world gen if you use that option 

again I should probably add better notes / output / description

**Reply by ElweThor** [reply]
> knarr only spawns on world gen if you use that option

So, what if I add this Mod after my map/world is started? (I bet that's the most probable scenario)

tbh: I was thinking at Knarr as a replacement of Better_Trader_Remake, 'cause that one is not working that well, right now, and I was seeing the last update was of time ago (AKA was not sure it was still updated/supported, even if not deprecated).
